### ISSUE TITLE
[Feature Request]: Addition of Fishers LSD as  ANOVA post-hoc test 

### ISSUE THREAD

**Issue by CJAB93** [jasp-stats/jasp-issues#475]
* Enhancement: 
I would love it if a Fishers Least Significant Difference test could be added as a post-hoc test after an ANOVA. 

* Purpose:
Now, if I use an ANOVA for a large number of different treatments/independent variables (which happens quite often in Biology), the quite conservative correction by Tukey etc makes it impossible to distinguish any trends: everything will be insignificant due to the high amount of pairwise comparisons. Using the non-(Bonferroni) corrected Fishers LSD can give leads on which treatment to investigate further, even if it is not (statistically speaking) correct/neat to interpret significant results as "real" significant results due to type I errors.

Also, it gives me, the researcher, the freedom to manually apply a Bonferroni-correction that I deem correct. Instead of a correction for ALL pairwise comparisons, I might only be interested in the comparisons between treatments and control, and not comparisons between two treatments. 

**Describe the solution you'd like**
I want Fishers LSD adopted in the post-hoc options.

Thanks!

**Comment by JohnnyDoorn** [comment]
Hi @CJAB93 

Thanks for that suggestion - this could also be the basis for showing a "grouping by letters" table. Would that also remedy your other issue? (https://github.com/jasp-stats/jasp-issues/issues/342)

Kind regards
Johnny

**Comment by danijela-i** [comment]
I would also love to see Fishers LSD added to the ANOVA post-hoc options. Is there any update on this?

**Comment by tomtomme** [comment]
@CJAB93 @danijela-i 
This is not forgotten. But it is not yet a priority. Sorry.

**Comment by JohnnyDoorn** [comment]
Hi @Ivo3185,

The next JASP release (out this week or the next) will have letter-based representations for post hoc tests, which I think is similar to Fishers LSD - is that something you could work with? If not, how does the use of LSD differ from the letter-based approach for you?

Cheers
Johnny

**Comment by Ivo3185** [comment]
Hello, @JohnnyDoorn,
Thank you so much for your quick reply. Letter-based representation would be a welcome addition, but would not resolve my need for Fisher's LSD. I want to present uncorrected _p_-values for each ANOVA pairwise comparison, the same way JASP allows me to do with BF10,U. Right now, you can only get corrected values, but those values are too conservative.

**Comment by Ivo3185** [comment]
> This issue will be automatically closed in 42 days due to inactivity. Feel free to leave a comment if you believe this is still relevant.

Still relevant

**Comment by R180** [comment]
Yes. For any kind of post-hoc tests (in ANOVA, LMM, etc.) there should be the option to have uncorrected p values (which in ANOVA are called Fisher's LSD), so that the user is free to then correct them (or not) according to a variety of methods. For example, when I have a set of uncorrected p values I can use p.adjust, in R, to apply a false-discovery-rate correction instead of a familywise-error correction. Currently, if I want to do this, I have to use some program other than JASP for my analyses.
